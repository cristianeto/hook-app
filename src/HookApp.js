import React from "react";

function HookApp(props) {
  return <h1>Hello World!</h1>;
}

export default HookApp;
