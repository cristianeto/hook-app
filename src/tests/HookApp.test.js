import React from "react";
import { shallow } from "enzyme";

import HookApp from "../HookApp";

describe("Testing to HookApp.js", () => {
  test("should be return the HookApp component", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
