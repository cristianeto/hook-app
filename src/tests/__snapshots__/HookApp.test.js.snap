// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing to HookApp.js should be return the HookApp component 1`] = `
<h1>
  Hello World!
</h1>
`;
